lib/ex_plasma/utxo.ex

```diff
@@ -229,4 +235,17 @@ defmodule ExPlasma.Utxo do
   defp truncate_leading_zero(<<0>>), do: <<0>>
   defp truncate_leading_zero(<<0>> <> binary), do: truncate_leading_zero(binary)
   defp truncate_leading_zero(binary), do: binary
+
+  # Validates that the Utxo is in the expected formats. Returns an error tuple
+  defp validate_output(%{owner: <<0::160>>}), do: {:error, {:owner, :cannot_be_zero}}
```

Comment:
not entirely sure how comprehensive the validation is intended to be at this stage. Is the correct size (`160`) and no-padding validated elsewhere?

Reply:
There's probably a few gaps I'm overlooking including #19 (I assume this is the no-padding validation? Let's open another PR). 

`160` should be the [correct size in accordance with elixir-omg](https://github.com/omisego/elixir-omg/blob/master/apps/omg/lib/omg/crypto.ex#L28).

Reply:
> 160 should be the correct size in accordance with elixir-omg.

precisely, I'm asking to wrap my head around, what's the intended location of this being validated (sorry to miss sth obvious here)